README.md

```diff
@@ -1,17 +1,33 @@
 # Drawn Together
 
+# Pre-requisites
+
+- Docker
+- nvm (optional)
+- node v21
+
 # Installation
 
 1. Clone repository
-  * `git clone https://github.com/icedtrees/drawn-together`
-2. Install Docker
+   1. `git clone https://github.com/icedtrees/drawn-together`
+2. `nvm use`
+3. `npm install` (optional - this will be done in the Docker container)
 
 # Running locally
 
-Using docker-compose:
+Using docker-compose for development:
+
+1. Run `docker compose up --build -d`
+   1. This will start a docker container for mongodb and one for your webserver, and install every dependency you need
+   2. This will run the backend in development mode with hot reload on both 
+      frontend and backend
+2. Access at http://localhost:8443/
+
+Using docker-compose to mimic production:
 
-1. Run `docker compose up --build`
-  * This will start a docker container for mongodb and one for your webserver, and install every dependency you need
+1. Run `docker compose --file compose-prod.yaml up --build -d`
+    1. This will start a docker container for mongodb and one for your webserver, and install every dependency you need
+    2. This will run the backend in production mode without hot reload
```

Comment:
ooc, when would you use this option in practice?

Reply:
I needed it to test that I wasn't majorly breaking the production deploy, before just pushing it to prod.

It's not 100% the same as prod deploy ofc, but it's much closer and at least lets you check basic correctness of the prod Dockerfile.

Reply:
makes sense to me. i suppose in theory we want some kind of automated testing to run this and do some health checks so we don't have to worry about it